src/Town/js/createTownName.ts

```diff
@@ -36,7 +41,7 @@ setup.createTownName = function (town) {
     // Replace existing names of buildings if they reference town name
     town.buildings.forEach(building => {
       building.name = building.name.replace(town.name, driftName)
-      building.tippyDescription = building.tippyDescription.replace(town.name, driftName)
+      if (building.tippyDescription) building.tippyDescription = building.tippyDescription.replace(town.name, driftName)
```

Comment:
Prefer to use a `{}`-block instead. One liners are not always bad, but they should only be used where the lines is extremely clear, such as a single short function call.